### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by r3nya**
Hello!

Just updated docs and fixed some issues with markdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Reporters provide information about the current collection run in a format that
   Specify one reporter name as `string` or provide more than one reporter name as a comma separated list of reporter names. Available reporters are: `cli`, `json`, `html` and `junit`.<br/><br/>
 Spaces should **not** be used between reporter names / commas whilst specifying a comma separted list of reporters. For instance:<br/><br/>
 :white_check_mark: `-r html,cli,json,junit` <br/>
-:x: `-r html, cli , json,junit`
+:x: `-r html, cli, json, junit`
```
Comment (kunagpal) [inline README.md]: @r3nya This is intentional, as it is meant to highlight an example of incorrectly using comma separated reporter names.

File: README.md [hunk 2]
```diff
@@ -319,7 +319,7 @@ return of the `newman.run` function is a run instance, which emits run events th
...
 | options.ignoreRedirects   | This specifies whether newman would automatically follow 3xx responses from servers.<br /><br />_Optional_<br />Type: `boolean`, Default value: `false` |
 | options.insecure          | Disables SSL verification checks and allows self-signed SSL certificates.<br /><br />_Optional_<br />Type: `boolean`, Default value: `false` |
-| options.bail              | A switch to specify whether or not to gracefully stop a collection run on encountering the first error. Takes additional modifiers as arguments to specify whether to end the run with an error for invalid name or path.<br /><br/>Available modifiers: `folder` and `failure`.<br />eg. `bail : ['folder']`<br /><br />_Optional_<br />Type: `boolean|object`, Default value: `false` |
+| options.bail              | A switch to specify whether or not to gracefully stop a collection run on encountering the first error. Takes additional modifiers as arguments to specify whether to end the run with an error for invalid name or path.<br /><br/>Available modifiers: `folder` and `failure`.<br />eg. `bail : ['folder']`<br /><br />_Optional_<br />Type: `boolean\|object`, Default value: `false` |
```
Comment (kunagpal) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (r3nya) [reply README.md]: @kunagpal fixed! :)

**Comment by r3nya** [comment]
Yay @kunagpal! :)

**Pull request merged**